### PR TITLE
[12.0][ADD] sale_secondary_salesperson: Sale Secondary Salesperson

### PR DESCRIPTION
**Pull request by victoralmau**
Add Sale Secondary Salesperson in form and report.

Please @joao-p-marques and @CarlosRoca13 can you review it?

@Tecnativa TT28664

### CODE REVIEW

File: sale_secondary_salesperson/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+There is no need to do anything to configure this module
```
Comment (CarlosRoca13) [inline sale_secondary_salesperson/readme/CONFIGURE.rst]: This file is not necessary if this module doesn't need any kind of configuration.

**Review by CarlosRoca13 — APPROVED** [review]
LGTM just a non blocking comment. You can add the .pot and es.po files :smile:

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: sale_secondary_salesperson/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Secondary Salesperson',
+    'category': 'Customer Relationship Management',
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/__manifest__.py]: This is not CRM, but sales.

File: sale_secondary_salesperson/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    'summary': """Sale Secondary Salesperson""",
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/__manifest__.py]: A summary with the same content of the name is useless. Remove it (and don't use triple quote for one liner string - except docstrings -).

File: sale_secondary_salesperson/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    'version': '12.0.1.0.1',
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/__manifest__.py]: 12.0.1.0.0

File: sale_secondary_salesperson/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    'license': 'AGPL-3',
+    'author': 'Tecnativa, Odoo Community Association (OCA)',
+    'maintainers': ['victoralmau'],
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/__manifest__.py]: Add development_status = "Production/Stable"

File: sale_secondary_salesperson/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Secondary Salesperson',
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/__manifest__.py]: Please change all these simple quotes to double quotes for avoiding a big change when migrating to 13.0

File: sale_secondary_salesperson/readme/USAGE.rst [hunk 7]
```diff
@@ -0,0 +1,3 @@
+* Go to Sales
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/readme/USAGE.rst]: Specify a bit more

File: sale_secondary_salesperson/readme/DESCRIPTION.rst [hunk 8]
```diff
@@ -0,0 +1 @@
+This module extends SALES to allow the definition of a secondary salesperson
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/readme/DESCRIPTION.rst]:
```suggestion
This module extends sales management to allow the definition of a secondary salesperson.
```

File: sale_secondary_salesperson/tests/test_sale_order.py [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+        cls.user_demo = cls.env.ref("base.user_demo")
+        cls.partner = cls.env.ref("base.res_partner_12")
+
+    def test_crm_with_distinct_secondary_user_id(self):
+        order = Form(self.env["sale.order"])
+        order.partner_id = self.partner
+        order.user_id = self.user_admin
+        order.secondary_user_id = self.user_demo
+        order = order.save()
+        self.assertEqual(order.secondary_user_id, self.user_demo)
```
Comment (pedrobaeza) [inline sale_secondary_salesperson/tests/test_sale_order.py]: This test is not adding value, as it's checking that ORM works, but no logic of this module.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I'm missing the automatic propagation of the second salesman in a module `sale_crm_secondary_salesperson`.

**Review by joao-p-marques — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
/ocabot merge nobump

Please create `crm_sale_secondary_salesperson`

**Pull request merged**